notification-channel-config-service-api/src/main/proto/org/hypertrace/notification/config/service/v1/notification_channel.proto

```diff
@@ -13,6 +13,13 @@ message NotificationChannelMutableData {
   string channel_name = 1;
   repeated EmailChannelConfig email_channel_config = 2;
   repeated WebhookChannelConfig webhook_channel_config = 3;
+  AwsS3BucketChannelConfig bucket_channel_config = 4;
+}
+
+message AwsS3BucketChannelConfig {
+  string role_arn = 1;
```

Comment:
`role_arn` is the arn of the role that would be assumed to push the data to bucket (since we have a secret free S3 integration via `WebIdentityToken`)

Reply:
Is this the only way we'll ever support authenticating to a bucket? If not, you may want to wrap the details in a one-of so we can support other mechanisms like a basic secret in the future.